doc/content/gateways/lora-basics-station/cups.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Configuration and Update Server Protocol"
+description: ""
+weight: -1
+---
+
+This page contains information about connecting your gateway to {{% tts %}} using the Configuration and Update Server (CUPS) protocol.
+
+<!--more-->
+
+## Requirements
+
+1. User account on {{% tts %}} with rights to create gateways.
+
+## Create a Gateway
+
+Follow instructions for [Adding Gateways in the Console]({{< ref "/getting-started/console/create-gateway" >}}) or [Adding Gateways Using the Command-line interface]({{< ref "/getting-started/cli/create-gateway" >}}).
+
+## Create an API Key
+
+CUPS requires an API Key with the following rights:
+- View gateway information
+- Edit basic gateway settings
+
+If you have not already created one, follow instructions for [Creating a Gateway API Key in the Console]({{< ref "/getting-started/console/create-gateway#create-gateway-api-key" >}}) or [Creating a Gateway API Key Using the Command-line interface]({{< ref "/getting-started/cli/create-gateway#create-gateway-api-key" >}}).
+
+## Configure Gateway
+
+In your gateway configuration, set the following fields:
+
+CUPS URI: `https://<server-address>:443`
+CUPS Trust: Amazon Root CA 1
```

Comment:
This is only for TTI CH and only currently. A better way is to run 
```
openssl s_client -connect <server-address>:443 -showcerts
```
And use the certs that are provided there.